tests/rule_based_profiler/parameter_builder/conftest.py

```diff
@@ -0,0 +1,184 @@
+    )
+    uniform_upper_quantile: Number = stats.uniform.ppf(
+        q=1 - false_positive_rate / 2, loc=uniform_lower_bound, scale=uniform_scale
+    )
+
+    bimodal_mean_1: int = 4000
+    bimodal_stdev_1: int = 500
+    bimodal_mean_2: int = 6000
+    bimodal_stdev_2: int = 500
+
+    # 10,000 samples taken from a bimodal mixture of equally weighted normal distributions.
+    # This approximation is being used due to the lack of functionality in scipy.stats regarding a
+    # percent point functipn (ppf) for gaussian mixtures.
+    # For a look at a similar statistical representation and implementation in R see:
+    # https://stats.stackexchange.com/questions/390931/compute-quantile-function-from-a-mixture-of-normal-distribution
+    bimodal_approximation = np.concatenate(
+        [
+            stats.norm.rvs(
+                loc=bimodal_mean_1,
+                scale=bimodal_stdev_1,
+                size=5000,
+            ),
+            stats.norm.rvs(
+                loc=bimodal_mean_2,
+                scale=bimodal_stdev_2,
+                size=5000,
+            ),
+        ]
+    )
+    bimodal_lower_quantile = np.quantile(
+        a=bimodal_approximation, q=false_positive_rate / 2
+    )
+    bimodal_upper_quantile = np.quantile(
+        a=bimodal_approximation, q=1 - false_positive_rate / 2
+    )
+
+    exponential_shape: int = 1.5
+    exponential_lower_bound: int = 5000
+    exponential_scale: int = 1000
+    exponential_lower_quantile: Number = stats.gamma.ppf(
+        q=false_positive_rate / 2,
+        a=exponential_shape,
+        loc=exponential_lower_bound,
+        scale=exponential_scale,
+    )
+    exponential_upper_quantile: Number = stats.gamma.ppf(
+        q=1 - false_positive_rate / 2,
+        a=exponential_shape,
+        loc=exponential_lower_bound,
+        scale=exponential_scale,
+    )
+
+    return {
+        "normal": {
+            "mean": normal_mean,
+            "stdev": normal_stdev,
+            "lower_quantile": normal_lower_quantile,
+            "upper_quantile": normal_upper_quantile,
+        },
+        "uniform": {
+            "lower_bound": uniform_lower_bound,
+            "scale": uniform_scale,
+            "lower_quantile": uniform_lower_quantile,
+            "upper_quantile": uniform_upper_quantile,
+        },
+        "bimodal": {
+            "mean_1": bimodal_mean_1,
+            "stdev_1": bimodal_stdev_1,
+            "mean_2": bimodal_mean_2,
+            "stdev_2": bimodal_stdev_2,
+            "lower_quantile": bimodal_lower_quantile,
+            "upper_quantile": bimodal_upper_quantile,
+        },
+        "exponential": {
+            "shape": exponential_shape,
+            "lower_bound": exponential_lower_bound,
+            "scale": exponential_scale,
+            "lower_quantile": exponential_lower_quantile,
+            "upper_quantile": exponential_upper_quantile,
+        },
+    }
+
+
+def _generate_distribution_samples(
```

Comment:
@NathanFarmer Any harm making this method public?  Thanks.

Reply:
@alexsherstinsky I see no harm in making these methods public, but for what purpose?

Reply:
@NathanFarmer For use by other classes/method -- seems like useful samples generator.  Thanks!